src/test/resources/log4j2.xml

```diff
@@ -1,20 +1,3 @@
-<!--
```

Comment:
Shouldn't we replace it by Orkes Copyright instead of just deleting?

Reply:
done